nemo/collections/asr/modules/conv_asr.py

```diff
@@ -69,15 +69,13 @@ def _prepare_for_export(self, **kwargs):
             if isinstance(m, MaskedConv1d):
                 m.use_mask = False
                 m_count += 1
-
-            if isinstance(m, MaskedConv1d):
                 if m.conv.stride[0] > 1 and 'mconv' in name:
                     stride = stride * m.conv.stride[0]
 
-            if isinstance(m, SqueezeExcite):
```

Comment:
This is necessary. Masked SE will not work without this when exported.

Reply:
Why, if the masks are taken care of from forward() ?